INFO - 2023-09-20 00:50:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 00:50:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 00:50:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 00:52:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 00:52:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 00:54:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 00:54:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 00:54:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-09-20 01:11:59 --> syntax error, unexpected token "$"
in APPPATH/Controllers/Compras.php on line 92.
 1 SYSTEMPATH/Autoloader/Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('/www/SistemaDeInventarios/app/Controllers/Compras.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Compras')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Compras')
 4 SYSTEMPATH/CodeIgniter.php(890): class_exists('\\App\\Controllers\\Compras', true)
 5 SYSTEMPATH/CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-20 01:12:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-09-20 01:18:12 --> Undefined array key "tco_doc4"
in APPPATH/Controllers/Compras.php on line 67.
 1 APPPATH/Controllers/Compras.php(67): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "tco_doc4"', '/www/SistemaDeInventarios/app/Controllers/Compras.php', 67)
 2 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 3 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-20 01:18:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-09-20 01:19:19 --> Undefined constant "App\Controllers\d"
in APPPATH/Controllers/Compras.php on line 70.
 1 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 2 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 01:19:49 --> move_uploaded_file(/opt/lampp/htdocs/app/Views/documentos/Ensayo - Grupo Cuarzo.pdf): Failed to open stream: No such file or directory
in APPPATH/Controllers/Compras.php on line 92.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'move_uploaded_file(/opt/lampp/htdocs/app/Views/documentos/Ensayo - Grupo Cuarzo.pdf): Failed to open stream: No such file or directory', '/www/SistemaDeInventarios/app/Controllers/Compras.php', 92)
 2 APPPATH/Controllers/Compras.php(92): move_uploaded_file('/opt/lampp/temp/php7zQnDU', '/opt/lampp/htdocs/app/Views/documentos/Ensayo - Grupo Cuarzo.pdf')
 3 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 4 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 5 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 01:25:32 --> Undefined variable $baseURL
in APPPATH/Controllers/Compras.php on line 72.
 1 APPPATH/Controllers/Compras.php(72): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $baseURL', '/www/SistemaDeInventarios/app/Controllers/Compras.php', 72)
 2 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 3 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 01:26:38 --> Undefined variable $base_url
in APPPATH/Controllers/Compras.php on line 72.
 1 APPPATH/Controllers/Compras.php(72): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $base_url', '/www/SistemaDeInventarios/app/Controllers/Compras.php', 72)
 2 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 3 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 01:27:08 --> move_uploaded_file(http://localhost:41062/www/SistemaDeInventarios//app/Views/documentos/20092023012708Ensayo - Grupo Cuarzo.pdf): Failed to open stream: HTTP wrapper does not support writeable connections
in APPPATH/Controllers/Compras.php on line 92.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'move_uploaded_file(http://localhost:41062/www/SistemaDeInventarios//app/Views/documentos/20092023012708Ensayo - Grupo Cuarzo.pdf): Failed to open stream: HTTP wrapper does not support writeable connections', '/www/SistemaDeInventarios/app/Controllers/Compras.php', 92)
 2 APPPATH/Controllers/Compras.php(92): move_uploaded_file('/opt/lampp/temp/phpjb8LkX', 'http://localhost:41062/www/SistemaDeInventarios//app/Views/documentos/20092023012708Ensayo - Grupo Cuarzo.pdf')
 3 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 4 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 5 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 01:35:55 --> move_uploaded_file(http://localhost:41062/www/SistemaDeInventarios/app/Views/documentos/20092023013555Ensayo - Grupo Cuarzo.pdf): Failed to open stream: HTTP wrapper does not support writeable connections
in APPPATH/Controllers/Compras.php on line 92.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'move_uploaded_file(http://localhost:41062/www/SistemaDeInventarios/app/Views/documentos/20092023013555Ensayo - Grupo Cuarzo.pdf): Failed to open stream: HTTP wrapper does not support writeable connections', '/www/SistemaDeInventarios/app/Controllers/Compras.php', 92)
 2 APPPATH/Controllers/Compras.php(92): move_uploaded_file('/opt/lampp/temp/phpFmOpvp', 'http://localhost:41062/www/SistemaDeInventarios/app/Views/documentos/20092023013555Ensayo - Grupo Cuarzo.pdf')
 3 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 4 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 5 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 01:41:12 --> move_uploaded_file(/opt/lampp/htdocs/app/Views/documentos/20092023014112Ensayo - Grupo Cuarzo.pdf): Failed to open stream: No such file or directory
in APPPATH/Controllers/Compras.php on line 92.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'move_uploaded_file(/opt/lampp/htdocs/app/Views/documentos/20092023014112Ensayo - Grupo Cuarzo.pdf): Failed to open stream: No such file or directory', '/www/SistemaDeInventarios/app/Controllers/Compras.php', 92)
 2 APPPATH/Controllers/Compras.php(92): move_uploaded_file('/opt/lampp/temp/php33slc8', '/opt/lampp/htdocs/app/Views/documentos/20092023014112Ensayo - Grupo Cuarzo.pdf')
 3 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 4 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 5 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 01:45:29 --> Undefined constant "App\Controllers\BASEPATH"
in APPPATH/Controllers/Compras.php on line 72.
 1 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 2 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 3 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 01:45:51 --> move_uploaded_file(/opt/lampp/htdocs/app/Views/documentos/20092023014551Ensayo - Grupo Cuarzo.pdf): Failed to open stream: No such file or directory
in APPPATH/Controllers/Compras.php on line 92.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'move_uploaded_file(/opt/lampp/htdocs/app/Views/documentos/20092023014551Ensayo - Grupo Cuarzo.pdf): Failed to open stream: No such file or directory', '/www/SistemaDeInventarios/app/Controllers/Compras.php', 92)
 2 APPPATH/Controllers/Compras.php(92): move_uploaded_file('/opt/lampp/temp/phpfH6JPU', '/opt/lampp/htdocs/app/Views/documentos/20092023014551Ensayo - Grupo Cuarzo.pdf')
 3 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 4 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 5 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 02:04:46 --> Undefined variable $tco_doc2
in APPPATH/Controllers/Compras.php on line 67.
 1 APPPATH/Controllers/Compras.php(67): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $tco_doc2', '/www/SistemaDeInventarios/app/Controllers/Compras.php', 67)
 2 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 3 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 02:20:08 --> syntax error, unexpected token "=", expecting "]"
in APPPATH/Controllers/Compras.php on line 72.
 1 SYSTEMPATH/Autoloader/Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('/www/SistemaDeInventarios/app/Controllers/Compras.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Compras')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Compras')
 4 SYSTEMPATH/CodeIgniter.php(890): class_exists('\\App\\Controllers\\Compras', true)
 5 SYSTEMPATH/CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 02:20:25 --> syntax error, unexpected variable "$this", expecting "]"
in APPPATH/Controllers/Compras.php on line 83.
 1 SYSTEMPATH/Autoloader/Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('/www/SistemaDeInventarios/app/Controllers/Compras.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Compras')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Compras')
 4 SYSTEMPATH/CodeIgniter.php(890): class_exists('\\App\\Controllers\\Compras', true)
 5 SYSTEMPATH/CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 02:20:38 --> Allowed fields must be specified for model: "App\Models\model_compras"
in SYSTEMPATH/BaseModel.php on line 1242.
 1 SYSTEMPATH/BaseModel.php(1242): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields('App\\Models\\model_compras')
 2 SYSTEMPATH/BaseModel.php(757): CodeIgniter\BaseModel->doProtectFields([...])
 3 SYSTEMPATH/Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 4 APPPATH/Controllers/Compras.php(89): CodeIgniter\Model->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 6 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 7 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 02:25:14 --> Allowed fields must be specified for model: "App\Models\model_compras"
in SYSTEMPATH/BaseModel.php on line 1242.
 1 SYSTEMPATH/BaseModel.php(1242): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields('App\\Models\\model_compras')
 2 SYSTEMPATH/BaseModel.php(757): CodeIgniter\BaseModel->doProtectFields([...])
 3 SYSTEMPATH/Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 4 APPPATH/Controllers/Compras.php(89): CodeIgniter\Model->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 6 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 7 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-20 02:27:25 --> mysqli_sql_exception: Column 'tco_cod_reglon' cannot be null in /www/SistemaDeInventarios/system/Database/MySQLi/Connection.php:295
Stack trace:
#0 /www/SistemaDeInventarios/system/Database/MySQLi/Connection.php(295): mysqli->query('INSERT INTO `tc...', 0)
#1 /www/SistemaDeInventarios/system/Database/BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tc...')
#2 /www/SistemaDeInventarios/system/Database/BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tc...')
#3 /www/SistemaDeInventarios/system/Database/BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tc...', Array, false)
#4 /www/SistemaDeInventarios/system/Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 /www/SistemaDeInventarios/system/BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 /www/SistemaDeInventarios/system/Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 /www/SistemaDeInventarios/app/Controllers/Compras.php(88): CodeIgniter\Model->insert(Array)
#8 /www/SistemaDeInventarios/system/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
#9 /www/SistemaDeInventarios/system/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
#10 /www/SistemaDeInventarios/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /www/SistemaDeInventarios/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-09-20 02:27:26 --> Column 'tco_cod_reglon' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 295.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(295): mysqli->query('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (\'ME-1234\', \'1\', \'2023-09-19\', \'Oficina Central\', \'12345678\', \'Central\', \'5\', \'ME-1234\', \'Ninguna\', \'Ninguno\', \'Proveedor 1\', NULL, \'12\', NULL, \'Ninguna Descripción\', \'100\', \'500\', \'Bernie\', \'Bernie\', \'Bernie\', \'Ninguno\', \'Ensayo - Grupo Cuarzo_2.pdf\', \'Infografía de 7 principios del testing_2.pdf\', \'Prueba de Software - JEST_2.pdf\', \'Resumen P1 Aseguramiento_2.pdf\', \'1\', \'1\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (\'ME-1234\', \'1\', \'2023-09-19\', \'Oficina Central\', \'12345678\', \'Central\', \'5\', \'ME-1234\', \'Ninguna\', \'Ninguno\', \'Proveedor 1\', NULL, \'12\', NULL, \'Ninguna Descripción\', \'100\', \'500\', \'Bernie\', \'Bernie\', \'Bernie\', \'Ninguno\', \'Ensayo - Grupo Cuarzo_2.pdf\', \'Infografía de 7 principios del testing_2.pdf\', \'Prueba de Software - JEST_2.pdf\', \'Resumen P1 Aseguramiento_2.pdf\', \'1\', \'1\')')
 3 SYSTEMPATH/Database/BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (\'ME-1234\', \'1\', \'2023-09-19\', \'Oficina Central\', \'12345678\', \'Central\', \'5\', \'ME-1234\', \'Ninguna\', \'Ninguno\', \'Proveedor 1\', NULL, \'12\', NULL, \'Ninguna Descripción\', \'100\', \'500\', \'Bernie\', \'Bernie\', \'Bernie\', \'Ninguno\', \'Ensayo - Grupo Cuarzo_2.pdf\', \'Infografía de 7 principios del testing_2.pdf\', \'Prueba de Software - JEST_2.pdf\', \'Resumen P1 Aseguramiento_2.pdf\', \'1\', \'1\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (:tco_cod_formulario:, :tco_version:, :tco_fecha_ingreso:, :tco_lugar:, :tco_numero:, :tco_unidad_admin:, :tco_cantidad:, :tco_formulario:, :tco_dependencia:, :tco_programa:, :tco_proveedor:, :tco_cod_reglon:, :tco_folio_almacen:, :tco_nomen_cuenta:, :tco_observacion:, :tco_valor:, :tco_valor_total:, :tco_Fnombre_almacen:, :tco_Fnombre_depto:, :tco_Fnombre_inventario:, :tco_ob_invetario:, :tco_doc1:, :tco_doc2:, :tco_doc3:, :tco_doc4:, :tco_idetr:, :tco_idper_registro:)', [...], false)
 5 SYSTEMPATH/Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Controllers/Compras.php(88): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
10 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
11 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-20 02:29:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 02:29:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-09-20 02:30:52 --> Allowed fields must be specified for model: "App\Models\model_compras"
in SYSTEMPATH/BaseModel.php on line 1242.
 1 SYSTEMPATH/BaseModel.php(1242): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields('App\\Models\\model_compras')
 2 SYSTEMPATH/BaseModel.php(757): CodeIgniter\BaseModel->doProtectFields([...])
 3 SYSTEMPATH/Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 4 APPPATH/Controllers/Compras.php(88): CodeIgniter\Model->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 6 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 7 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-20 02:31:54 --> mysqli_sql_exception: Column 'tco_nomen_cuenta' cannot be null in /www/SistemaDeInventarios/system/Database/MySQLi/Connection.php:295
Stack trace:
#0 /www/SistemaDeInventarios/system/Database/MySQLi/Connection.php(295): mysqli->query('INSERT INTO `tc...', 0)
#1 /www/SistemaDeInventarios/system/Database/BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tc...')
#2 /www/SistemaDeInventarios/system/Database/BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tc...')
#3 /www/SistemaDeInventarios/system/Database/BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tc...', Array, false)
#4 /www/SistemaDeInventarios/system/Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 /www/SistemaDeInventarios/system/BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 /www/SistemaDeInventarios/system/Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 /www/SistemaDeInventarios/app/Controllers/Compras.php(88): CodeIgniter\Model->insert(Array)
#8 /www/SistemaDeInventarios/system/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
#9 /www/SistemaDeInventarios/system/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
#10 /www/SistemaDeInventarios/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /www/SistemaDeInventarios/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-09-20 02:31:54 --> Column 'tco_nomen_cuenta' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 295.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(295): mysqli->query('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (\'ME-1234\', \'1\', \'2023-09-19\', \'Oficina Central\', \'12345678\', \'Central\', \'5\', \'ME-1234\', \'Ninguna\', \'Ninguno\', \'Proveedor 1\', \'1\', \'1\', NULL, \'Ninguna\', \'100\', \'500\', \'Bernie\', \'Bernie\', \'Bernie\', \'Ninguna\', \'Ensayo - Grupo Cuarzo_1.pdf\', \'Infografía de 7 principios del testing_1.pdf\', \'Prueba de Software - JEST_1.pdf\', \'Resumen P1 Aseguramiento_1.pdf\', \'1\', \'1\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (\'ME-1234\', \'1\', \'2023-09-19\', \'Oficina Central\', \'12345678\', \'Central\', \'5\', \'ME-1234\', \'Ninguna\', \'Ninguno\', \'Proveedor 1\', \'1\', \'1\', NULL, \'Ninguna\', \'100\', \'500\', \'Bernie\', \'Bernie\', \'Bernie\', \'Ninguna\', \'Ensayo - Grupo Cuarzo_1.pdf\', \'Infografía de 7 principios del testing_1.pdf\', \'Prueba de Software - JEST_1.pdf\', \'Resumen P1 Aseguramiento_1.pdf\', \'1\', \'1\')')
 3 SYSTEMPATH/Database/BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (\'ME-1234\', \'1\', \'2023-09-19\', \'Oficina Central\', \'12345678\', \'Central\', \'5\', \'ME-1234\', \'Ninguna\', \'Ninguno\', \'Proveedor 1\', \'1\', \'1\', NULL, \'Ninguna\', \'100\', \'500\', \'Bernie\', \'Bernie\', \'Bernie\', \'Ninguna\', \'Ensayo - Grupo Cuarzo_1.pdf\', \'Infografía de 7 principios del testing_1.pdf\', \'Prueba de Software - JEST_1.pdf\', \'Resumen P1 Aseguramiento_1.pdf\', \'1\', \'1\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (:tco_cod_formulario:, :tco_version:, :tco_fecha_ingreso:, :tco_lugar:, :tco_numero:, :tco_unidad_admin:, :tco_cantidad:, :tco_formulario:, :tco_dependencia:, :tco_programa:, :tco_proveedor:, :tco_cod_reglon:, :tco_folio_almacen:, :tco_nomen_cuenta:, :tco_observacion:, :tco_valor:, :tco_valor_total:, :tco_Fnombre_almacen:, :tco_Fnombre_depto:, :tco_Fnombre_inventario:, :tco_ob_invetario:, :tco_doc1:, :tco_doc2:, :tco_doc3:, :tco_doc4:, :tco_idetr:, :tco_idper_registro:)', [...], false)
 5 SYSTEMPATH/Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Controllers/Compras.php(88): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
10 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
11 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-20 02:32:56 --> Allowed fields must be specified for model: "App\Models\model_compras"
in SYSTEMPATH/BaseModel.php on line 1242.
 1 SYSTEMPATH/BaseModel.php(1242): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields('App\\Models\\model_compras')
 2 SYSTEMPATH/BaseModel.php(757): CodeIgniter\BaseModel->doProtectFields([...])
 3 SYSTEMPATH/Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 4 APPPATH/Controllers/Compras.php(88): CodeIgniter\Model->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 6 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 7 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-20 02:33:07 --> mysqli_sql_exception: Column 'tco_nomen_cuenta' cannot be null in /www/SistemaDeInventarios/system/Database/MySQLi/Connection.php:295
Stack trace:
#0 /www/SistemaDeInventarios/system/Database/MySQLi/Connection.php(295): mysqli->query('INSERT INTO `tc...', 0)
#1 /www/SistemaDeInventarios/system/Database/BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tc...')
#2 /www/SistemaDeInventarios/system/Database/BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tc...')
#3 /www/SistemaDeInventarios/system/Database/BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tc...', Array, false)
#4 /www/SistemaDeInventarios/system/Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 /www/SistemaDeInventarios/system/BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 /www/SistemaDeInventarios/system/Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 /www/SistemaDeInventarios/app/Controllers/Compras.php(88): CodeIgniter\Model->insert(Array)
#8 /www/SistemaDeInventarios/system/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
#9 /www/SistemaDeInventarios/system/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
#10 /www/SistemaDeInventarios/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /www/SistemaDeInventarios/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-09-20 02:33:07 --> Column 'tco_nomen_cuenta' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 295.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(295): mysqli->query('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (\'ME-1234\', \'1\', \'2023-09-19\', \'Oficina Central\', \'12345678\', \'Central\', \'5\', \'ME-1234\', \'Ninguna\', \'Ninguno\', \'Proveedor 1\', \'1\', \'1\', NULL, \'Ninguna\', \'100\', \'500\', \'Bernie\', \'Bernie\', \'Bernie\', \'Ninguna\', \'Ensayo - Grupo Cuarzo_3.pdf\', \'Infografía de 7 principios del testing_3.pdf\', \'Prueba de Software - JEST_3.pdf\', \'Resumen P1 Aseguramiento_3.pdf\', \'1\', \'1\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (\'ME-1234\', \'1\', \'2023-09-19\', \'Oficina Central\', \'12345678\', \'Central\', \'5\', \'ME-1234\', \'Ninguna\', \'Ninguno\', \'Proveedor 1\', \'1\', \'1\', NULL, \'Ninguna\', \'100\', \'500\', \'Bernie\', \'Bernie\', \'Bernie\', \'Ninguna\', \'Ensayo - Grupo Cuarzo_3.pdf\', \'Infografía de 7 principios del testing_3.pdf\', \'Prueba de Software - JEST_3.pdf\', \'Resumen P1 Aseguramiento_3.pdf\', \'1\', \'1\')')
 3 SYSTEMPATH/Database/BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (\'ME-1234\', \'1\', \'2023-09-19\', \'Oficina Central\', \'12345678\', \'Central\', \'5\', \'ME-1234\', \'Ninguna\', \'Ninguno\', \'Proveedor 1\', \'1\', \'1\', NULL, \'Ninguna\', \'100\', \'500\', \'Bernie\', \'Bernie\', \'Bernie\', \'Ninguna\', \'Ensayo - Grupo Cuarzo_3.pdf\', \'Infografía de 7 principios del testing_3.pdf\', \'Prueba de Software - JEST_3.pdf\', \'Resumen P1 Aseguramiento_3.pdf\', \'1\', \'1\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tco_transaccion_compra` (`tco_cod_formulario`, `tco_version`, `tco_fecha_ingreso`, `tco_lugar`, `tco_numero`, `tco_unidad_admin`, `tco_cantidad`, `tco_formulario`, `tco_dependencia`, `tco_programa`, `tco_proveedor`, `tco_cod_reglon`, `tco_folio_almacen`, `tco_nomen_cuenta`, `tco_observacion`, `tco_valor`, `tco_valor_total`, `tco_Fnombre_almacen`, `tco_Fnombre_depto`, `tco_Fnombre_inventario`, `tco_ob_invetario`, `tco_doc1`, `tco_doc2`, `tco_doc3`, `tco_doc4`, `tco_idetr`, `tco_idper_registro`) VALUES (:tco_cod_formulario:, :tco_version:, :tco_fecha_ingreso:, :tco_lugar:, :tco_numero:, :tco_unidad_admin:, :tco_cantidad:, :tco_formulario:, :tco_dependencia:, :tco_programa:, :tco_proveedor:, :tco_cod_reglon:, :tco_folio_almacen:, :tco_nomen_cuenta:, :tco_observacion:, :tco_valor:, :tco_valor_total:, :tco_Fnombre_almacen:, :tco_Fnombre_depto:, :tco_Fnombre_inventario:, :tco_ob_invetario:, :tco_doc1:, :tco_doc2:, :tco_doc3:, :tco_doc4:, :tco_idetr:, :tco_idper_registro:)', [...], false)
 5 SYSTEMPATH/Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Controllers/Compras.php(88): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
10 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
11 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-20 02:33:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 02:33:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-09-20 02:34:28 --> The route for "ListarComprar" cannot be found.
in SYSTEMPATH/HTTP/RedirectResponse.php on line 60.
 1 SYSTEMPATH/HTTP/RedirectResponse.php(60): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('ListarComprar')
 2 APPPATH/Controllers/Compras.php(93): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 SYSTEMPATH/CodeIgniter.php(932): App\Controllers\Compras->registrarcompra()
 4 SYSTEMPATH/CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Compras))
 5 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-20 02:36:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 02:36:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 02:37:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 02:39:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 02:39:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 03:06:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 03:06:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 03:06:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 03:07:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 03:07:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 03:07:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 03:07:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 22:52:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 22:52:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 22:53:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-20 22:53:24 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\SistemaDeInventarios\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\SistemaDeInventarios\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'inventario_inab', 3306, '', 0)
#1 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\SistemaDeInventarios\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\SistemaDeInventarios\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\SistemaDeInventarios\app\Models\Model_Login.php(15): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\SistemaDeInventarios\app\Controllers\Login.php(28): App\Models\Model_Login->getUserBy('per_correo', 'manuel@manuel.c...')
#8 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(932): App\Controllers\Login->signin()
#9 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\SistemaDeInventarios\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\SistemaDeInventarios\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\SistemaDeInventarios\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\SistemaDeInventarios\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\SistemaDeInventarios\app\Models\Model_Login.php(15): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\SistemaDeInventarios\app\Controllers\Login.php(28): App\Models\Model_Login->getUserBy('per_correo', 'manuel@manuel.c...')
#7 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(932): App\Controllers\Login->signin()
#8 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\SistemaDeInventarios\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-09-20 22:53:24 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `per_personal`
WHERE `per_correo` = :per_correo:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\Model_Login.php(15): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Login.php(28): App\Models\Model_Login->getUserBy('per_correo', 'manuel@manuel.com')
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Login->signin()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-20 22:53:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-20 22:53:31 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\SistemaDeInventarios\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\SistemaDeInventarios\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'inventario_inab', 3306, '', 0)
#1 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\SistemaDeInventarios\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\SistemaDeInventarios\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\SistemaDeInventarios\app\Models\Model_Login.php(15): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\SistemaDeInventarios\app\Controllers\Login.php(28): App\Models\Model_Login->getUserBy('per_correo', 'manuel@manuel.c...')
#8 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(932): App\Controllers\Login->signin()
#9 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\SistemaDeInventarios\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\SistemaDeInventarios\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\SistemaDeInventarios\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\SistemaDeInventarios\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\SistemaDeInventarios\app\Models\Model_Login.php(15): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\SistemaDeInventarios\app\Controllers\Login.php(28): App\Models\Model_Login->getUserBy('per_correo', 'manuel@manuel.c...')
#7 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(932): App\Controllers\Login->signin()
#8 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\SistemaDeInventarios\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-09-20 22:53:31 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `per_personal`
WHERE `per_correo` = :per_correo:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\Model_Login.php(15): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Login.php(28): App\Models\Model_Login->getUserBy('per_correo', 'manuel@manuel.com')
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Login->signin()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-20 22:53:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-20 23:46:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\SistemaDeInventarios\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\SistemaDeInventarios\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'inventario_inab', 3306, '', 0)
#1 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\SistemaDeInventarios\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\SistemaDeInventarios\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\SistemaDeInventarios\app\Models\Model_Login.php(15): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\SistemaDeInventarios\app\Controllers\Login.php(28): App\Models\Model_Login->getUserBy('per_correo', 'manuel@manuel.c...')
#8 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(932): App\Controllers\Login->signin()
#9 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\SistemaDeInventarios\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\SistemaDeInventarios\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\SistemaDeInventarios\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\SistemaDeInventarios\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\SistemaDeInventarios\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\SistemaDeInventarios\app\Models\Model_Login.php(15): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\SistemaDeInventarios\app\Controllers\Login.php(28): App\Models\Model_Login->getUserBy('per_correo', 'manuel@manuel.c...')
#7 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(932): App\Controllers\Login->signin()
#8 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\SistemaDeInventarios\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\SistemaDeInventarios\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-09-20 23:46:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `per_personal`
WHERE `per_correo` = :per_correo:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\Model_Login.php(15): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Login.php(28): App\Models\Model_Login->getUserBy('per_correo', 'manuel@manuel.com')
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Login->signin()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-20 23:46:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 23:48:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 23:48:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 23:48:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 23:48:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 23:48:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 23:48:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-20 23:48:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
